modules/websearch/lib/websearch_templates.py

```diff
@@ -167,6 +168,9 @@ class Template:
         'wl': (int, CFG_WEBSEARCH_WILDCARD_LIMIT),
         }
 
+    if CFG_WEBSEARCH_OBELIX_REDIS:
+        search_results_default_urlargd['rm'] = (str, "rrm")
```

Comment:
If you can check the other comments I made would be nice.
But I think we should put `rrm` as default outside of your work, meaning on the overlay, so we have full control (better control) of what is going on depending on which collection users are searching on.

We should also think about a redesign of the search box to include how the results are ranked, so your work is correctly advertised, now it is not very clear. 
Maybe you can also include this UI problem on your research results ;-)